sklearn/base.py

```diff
@@ -252,6 +254,61 @@ def get_params(self, deep=True):
             out[key] = value
         return out
 
+    def _get_params_html(self, deep=True):
+        """
+        Get parameters for this estimator with a specific HTML representation.
+
+        Parameters
+        ----------
+        deep : bool, default=True
+            If True, will return the parameters for this estimator and
+            contained subobjects that are estimators.
+
+        Returns
+        -------
+        params : ParamsDict
+            Parameter names mapped to their values. We return a `ParamsDict`
+            dictionary, which renders a specific HTML representation in table
+            form.
+        """
+        out = self.get_params(deep=deep)
+
```

Comment:
> I'll start a list of feedback that are not related to the code directly and I'll update it as I'm reviewing the PR:
> 
> * When showing the parameters in the table, I think that we should keep the same order as in the constructor signature. Right now, I'm under the impression that we order them using alphabetic order.

The order of the parameters comes from `get_params` which in turn uses `_get_param_names`, and this orders the parameters names - unless I'm missing something. So should I implement a new `get_param` just for our table @glemaitre ?

Reply:
I would prefer to reorder them based on the signature of the `__init__` or function if possible. We can let `get_params` as is but modify `out`.